src/TheThingsNetwork.cpp

```diff
@@ -345,12 +345,12 @@ void TheThingsNetwork::clearReadBuffer()
   }
 }
 
-size_t TheThingsNetwork::readLine(char *buffer, size_t size, uint8_t timeout)
+size_t TheThingsNetwork::readLine(char *buffer, size_t size, uint8_t attempts)
 {
   size_t read = 0;
-  while (read == 0 && timeout > 0)
+  while (read == 0 && attempts > 0)
   {
-    timeout--;
```

Comment:
`readBytesUntil` has a timeout of 1000ms. If you have multiple attempts to call that function, the attempts are directly proportional to the timeout. `timeout` makes more sense to me, because it is a value in seconds.